plugins/fullpage.md

```diff
@@ -2,14 +2,14 @@
 layout: default
 title: Full Page Plugin
 title_nav: Full Page
-description: Edit all document properties such as title, keywords and description.
+description: Edit all Metadata and document properties such as title, keywords and description.
```

Comment:
Probably shouldn't be a capital M here, since we're not referring to the actual dialog/menu name but more of a general concept.